geometry_msgs/Point point
geometry_msgs/Vector3 norm_vec
